frontend/talentsearch/src/js/components/pool/PoolAdvertisementPage.tsx

```diff
@@ -0,0 +1,699 @@
+import React from "react";
+import { useIntl } from "react-intl";
+import { BriefcaseIcon } from "@heroicons/react/solid";
+
+import Breadcrumbs from "@common/components/Breadcrumbs";
+import type { BreadcrumbsProps } from "@common/components/Breadcrumbs";
+import NotFound from "@common/components/NotFound";
+import Pending from "@common/components/Pending";
+import Card from "@common/components/Card";
+import { Link } from "@common/components";
+import { getLocale, getLocalizedName } from "@common/helpers/localize";
+import { imageUrl } from "@common/helpers/router";
+
+import { AdvertisementStatus, SkillCategory } from "@common/api/generated";
+import TableOfContents from "@common/components/TableOfContents";
+import {
+  LightningBoltIcon,
+  BriefcaseIcon as BriefcaseIconOutline,
+  PhoneIcon,
+  LightBulbIcon,
+  CheckCircleIcon,
+  ChipIcon,
+  CloudIcon,
+} from "@heroicons/react/outline";
+import Accordion from "@common/components/accordion";
+import { strong } from "@common/helpers/format";
+import {
+  getLanguageRequirement,
+  getSecurityClearance,
+} from "@common/constants/localizedConstants";
+import { categorizeSkill } from "@common/helpers/skillUtils";
+import { Role, useGetPoolAdvertisementQuery } from "../../api/generated";
+import type { PoolAdvertisement } from "../../api/generated";
+import { useDirectIntakeRoutes } from "../../directIntakeRoutes";
+import TALENTSEARCH_APP_DIR from "../../talentSearchConstants";
+import commonMessages from "../commonMessages";
+import PoolInfoCard from "./PoolInfoCard";
+import ClassificationDefinition from "../ClassificationDefinition/ClassificationDefinition";
+
+interface ApplyButtonProps {
+  disabled: boolean;
+  href: string;
+}
+
+const ApplyButton = ({ disabled, href }: ApplyButtonProps) => {
+  const intl = useIntl();
+  return (
+    <Link
+      type="button"
+      color="primary"
+      mode="solid"
+      disabled={disabled}
+      href={href}
+    >
+      {intl.formatMessage({
+        defaultMessage: "Apply for this process",
+        description: "Link text to apply for a pool advertisement",
+      })}
+    </Link>
+  );
+};
+interface IconTitleProps {
+  children: React.ReactNode;
+  icon: React.FC<React.SVGProps<SVGSVGElement>>;
+}
+
+const IconTitle = ({ children, icon }: IconTitleProps) => {
+  const Icon = icon;
+
+  return (
+    <h3
+      data-h2-display="b(flex)"
+      data-h2-align-items="b(center)"
+      data-h2-font-size="b(h4)"
+    >
+      <Icon style={{ width: "1em", marginRight: "0.5rem" }} />
+      <span>{children}</span>
+    </h3>
+  );
+};
+
+// NOTE: Not entirely sure why this is failing?
+const accommodationEmail = (chunks: string[]) => (
+  // eslint-disable-next-line jsx-a11y/anchor-has-content
+  <a href="mailto:fames@acanteipsum.ca">{...chunks}</a>
+);
```

Comment:
I have done linking like this
https://github.com/GCTC-NTGC/gc-digital-talent/blob/main/frontend/talentsearch/src/js/components/roleSalaryForm/RoleSalaryForm.tsx#L93
then 
https://github.com/GCTC-NTGC/gc-digital-talent/blob/main/frontend/talentsearch/src/js/components/roleSalaryForm/RoleSalaryForm.tsx#L263

Reply:
🤷 
the `msg: string` `{msg}` way I linked throws no linting issue so I dunno, maybe the linter doesn't recognize your method